modules/permutiveRtdProvider.js

```diff
@@ -260,6 +268,39 @@ function readSegments (key) {
   }
 }
 
+/**
+ * Bidder-specific extension data for use in the ORTB2 object.
+ */
+const extensionDataByBidder = {
+  ix: {
+    segtax: '4'
+  }
+}
+
+const unknownIabSegmentId = '_unknown_'
+
+/**
+ * Bidder-specific functions to apply to an array of segment IDs.
+ * Each function should return an a new array of transformed segment IDs.
+ */
+const segmentIdTransformationsByBidder = {
+  ix: segments => segments.map(iabSegmentId).filter(id => id != unknownIabSegmentId)
```

Comment:
Intellij is telling me: `Comparison id != unknownIabSegmentId may cause unexpected type coercion`

Reply:
Does it go away if you change to `!==`?

Reply:
yep 👍